build/scripts/cibuild.ps1

```diff
@@ -52,6 +52,24 @@ function Terminate-BuildProcesses() {
     Get-Process vbcscompiler -ErrorAction SilentlyContinue | kill
 }
 
+# Ensure that procdump is available on the machine.  Returns the path to the directory that contains 
+# the procdump binaries (both 32 and 64 bit)
+function Ensure-ProcDump() {
```

Comment:
Actually ... why do you think that?  The Jenkins machine are simply Win2K16 machines now in many cases.  We took a bare image specifically to avoid depending on any special tools.